num_to_words/exception.py

```diff
@@ -59,4 +59,6 @@ def occurs_at_end(piece):
         for expt in exception_dict:
             if occurs_at_end(expt):
                 words = words.replace(expt, exception_dict[expt])
+    elif lang == "ta":
```

Comment:
I see your point but I would keep things simple.

There are two school of though here.
In modern usage, the Tamil language predominantly uses the Hindu-Arabic numeral system, where "௧௦" (the combination of 1 and 0) is used to represent the number 10. The traditional Tamil numeral "௰" for 10 is less commonly used today and is primarily found in historical texts, inscriptions, and certain cultural or ceremonial contexts.

So, "௧௦" is more common than "௰" in contemporary usage.
(I might be wrong with my limited tamil knowledge, please attach some source if you think otherwise)

Reply:
I don't have an opinion there. I thought the digit symbol ten `௰` should be handled anyway since it exists.